backend/src/test/java/com/woowacourse/momo/auth/service/AuthServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package com.woowacourse.momo.auth.service;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.woowacourse.momo.auth.dto.LoginRequest;
+import com.woowacourse.momo.auth.exception.AuthFailException;
+import com.woowacourse.momo.member.dto.request.SignUpRequest;
+import com.woowacourse.momo.member.service.MemberService;
+
+@Transactional
+@SpringBootTest
+class AuthServiceTest {
+
+    private static final String EMAIL = "woowa@woowa.com";
+    private static final String PASSWORD = "wooteco1!";
+
+    @Autowired
+    private AuthService authService;
+
+    @Autowired
+    private MemberService memberService;
+
+    @BeforeEach
+    void setUp() {
+        SignUpRequest request = new SignUpRequest(EMAIL, PASSWORD, "모모");
+        memberService.signUp(request);
+    }
+
+    @DisplayName("로그인을 성공한다")
+    @Test
+    void signIn() {
+        LoginRequest request = new LoginRequest(EMAIL, PASSWORD);
+
+        assertDoesNotThrow(() -> authService.login(request));
```

Comment:
```suggestion
        assertThat(authService.login(request).getAccessToken()).isNotNull();
```
이런 식으로 토큰에 대한 정보를 확인하는 테스트가 더 정확할 것 같아요.